home.html

```diff
@@ -35,16 +35,59 @@
           </ul>
         </div>
         <i class="fa fa-bars" onclick="showMenu()"></i>
-        <!-- Building the burger icon below-->
-        <!-- <i class="fa fa-bars"></i> -->
       </nav>
-      <div class="text-box">
+      <!-- Slideshow container -->
+      <div class="slideshow-container">
+        <!-- Full-width images with number and caption text -->
+        <div class="mySlides fade">
+          <div class="numbertext">1 / 3</div>
```

Comment:
@AmaniLouendriz I would suggest separating the words here `<div class="number-text">1 / 3</div>`